addons/hr_expense/models/hr_expense.py

```diff
@@ -179,7 +179,7 @@ def _default_employee_id(self):
         comodel_name='account.account',
         string="Account",
         compute='_compute_account_id', precompute=True, store=True, readonly=False,
-        domain="[('account_type', 'not in', ('asset_receivable', 'liability_payable', 'asset_cash', 'liability_credit_card')), ('company_id', '=', company_id)]",
+        domain="[('account_type', 'not in', ('asset_receivable', 'liability_payable', 'asset_cash', 'liability_credit_card'))]",
```

Comment:
You could add `check_company=True` - that way, the account will be constrained to one that is accessible from the current company.

Otherwise, if the user has several companies active, they might be able to select accounts that don't belong to the company of the expense or its parents.

(I discussed with @JulienAlardot that this is indeed something we want on the hr.expense model)

Thanks!

Reply:
Done! Thanks for taking the time to look into this.